### PR TITLE
install role: use GitHub token if available

### PR DESCRIPTION
**Pull request by felixfontein**
Was part of #131, which exposed a bug in ansible-core.

### CODE REVIEW

File: roles/install/meta/argument_specs.yml [hunk 1]
```diff
@@ -51,3 +51,10 @@ argument_specs:
         description:
           - 'Whether the role should use C(become: true) when installing packages.'
         type: bool
+      sops_github_token:
+        description:
+          - Token to provide when querying the GitHub API for the latest release. Without the token
+            there are rather strict rate limits.
+          - Should mainly be used in GitHub Actions.
+        type: string
```
Comment (kristianheljas) [inline roles/install/meta/argument_specs.yml:59]: This is what caused the weird string conversion error, argument_spec expects `type: str`

Comment (kristianheljas) [inline roles/install/meta/argument_specs.yml:59]:
```suggestion
        type: str
```

**Review by kristianheljas — APPROVED** [review]
LGTM!

**Comment by felixfontein** [comment]
@kristianheljas thanks for reviewing this!

**Pull request merged**